src/beyondllm/llms/together_ai.py

```diff
@@ -0,0 +1,66 @@
+from beyondllm.llms.base import BaseLLMModel, ModelConfig
+from typing import Any, Dict
+from dataclasses import dataclass, field
+import os
+from together import Together
```

Comment:
import the together module inside ``load_llm`` refer to other LLMs. This needs to be under exception block